keras_cv/models/object_detection/retinanet/retinanet.py

```diff
@@ -414,37 +419,37 @@ def compute_loss(self, x, box_pred, cls_pred, boxes, classes):
                 "parameter?"
             )
 
-        cls_labels = tf.one_hot(
-            tf.cast(classes, dtype=tf.int32),
-            depth=self.num_classes,
-            dtype=tf.float32,
+        # Torch one_hot doesn't support negative inputs, so we get rid of
```

Comment:
This sounds like something that should be fixed at the level of `keras_core.backend.torch.nn.one_hot`, not here.

Reply:
Good call -- I've opened https://github.com/keras-team/keras-core/pull/427 to fix this in keras core, and I'll update this once that PR is merged.

Reply:
I've gone ahead and updated this here, so CI will be red until the upstream PR is merged.